lms/djangoapps/grades/docs/how_tos/how-to-regrade-everyone-in-a-course.rst

```diff
@@ -8,27 +8,18 @@ grades for every learner in a course.
 Steps to trigger course-wide re-grade
 =====================================
 
-1. Go to the Studio grading settings for the course,
-   for example https://studio.stage.edx.org/settings/grading/edX/DemoX/Demo_Course .
+1. Go to LMS Django Admin and add an entry in ComputeGradesSetting, in the Grades section. 
+   https://courses-internal.edx.org/admin/grades/computegradessetting/
```

Comment:
Lets obscure this URL a bit.  Maybe just `/admin/grades/computegradessetting/`

Reply:
In the "What to do if grades are frozen" we directly link to a stage admin page. Should I change that too?